sfa_dash/blueprints/auth0.py

```diff
@@ -0,0 +1,63 @@
+from functools import partial
+
+
+try:
+    from flask import _app_ctx_stack as stack
+except ImportError:
+    from flask import _request_ctx_stack as stack
+from flask import url_for, redirect, current_app, session
+from flask.globals import LocalProxy, _lookup_app_object
+from flask_dance.consumer import OAuth2ConsumerBlueprint
+from six.moves.urllib.parse import urlencode
+
+
+auth0 = LocalProxy(partial(_lookup_app_object, 'auth0_oauth'))
```

Comment:
Although this is how the flask dance examples do it, naming the session for the Oauth provider was kind of confusing here, and it even gets renamed to session in the api interface. Maybe just call it auth0_session or something similar?